dictionaries/CallMap.php

```diff
@@ -8447,6 +8447,7 @@
 'mysqli_fetch_all' => ['array', 'result'=>'mysqli_result', 'mode='=>'int'],
 'mysqli_fetch_array' => ['?array', 'result'=>'mysqli_result', 'mode='=>'int'],
 'mysqli_fetch_assoc' => ['array<string,string>|null', 'result'=>'mysqli_result'],
+'mysqli_fetch_column' => ['null|int|float|string|false', 'result'=>'mysqli_result', 'column'=>'int'],
```

Comment:
Column is optional, so it needs to be marked as such by postfixing the name with `=`
```suggestion
'mysqli_fetch_column' => ['null|int|float|string|false', 'result'=>'mysqli_result', 'column='=>'int'],
```